scripts/install_rln_tests_dependencies.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Install Anvil
+if ! command -v anvil &> /dev/null; then
+    ./scripts/install_anvil.sh
+fi
+
+#Install pnpm
+if ! command -v pnpm &> /dev/null; then
+    ./scripts/install_pnpm.sh
+fi
```

Comment:
Maybe not needed? Similarly for the anvil's script
```suggestion
./scripts/install_pnpm.sh
```

Reply:
Added the if check in https://github.com/waku-org/nwaku/pull/3408/commits/30f288f0950f3b55662d4c9482cffc7a3e01785a